src/ladds/particle/Constellation.cpp

```diff
@@ -14,27 +14,16 @@
 
 std::mt19937 Constellation::generator{42};
 
-Constellation::Constellation(const std::string &constellation_data_str,
+Constellation::Constellation(const YAML::Node &constellationConfig,
```

Comment:
it works, the ConfigReader that is passed contains the path to the .yaml file for the individual constellation, which is represented by a `YAML::Node` currently. I will change the implementation to use the wrapper class instead of the node, so the code is more consistent